bindings/rust/wasmedge-sdk/Cargo.toml

```diff
@@ -14,7 +14,7 @@ version = "0.1.0"
 anyhow = "1.0"
 thiserror = "1.0.30"
 wasmedge-sys = {path = "../wasmedge-sys", version = "0.7"}
-wasmedge-types = "0.1.2"
+wasmedge-types = "0.1"
```

Comment:
Why do you downgrade the version?

Reply:
Not exactly. `wasmedge-types = "0.1"` means that the latest `0.1.x` will be used. In this case, for example, `0.1.3` will be used. This way is much more flexible. Thanks!